tools/fix_auth/fix_auth.rb

```diff
@@ -84,8 +84,6 @@ def load_rails
 
     def set_passwords
       ManageIQ::Password.key_root = cert_dir if cert_dir
-      ManageIQ::Password.add_legacy_key("v0_key", :v0)
-      ManageIQ::Password.add_legacy_key("v1_key", :v1)
       if options[:legacy_key] && !ManageIQ::Password.add_legacy_key(options[:legacy_key])
```

Comment:
do you want to keep this or change to the key["aux"] format you use in the tests?